lib-multisrc/mmlook/src/eu/kanade/tachiyomi/multisrc/mmlook/MMLook.kt

```diff
@@ -0,0 +1,200 @@
+package eu.kanade.tachiyomi.multisrc.mmlook
+
+import eu.kanade.tachiyomi.lib.unpacker.Unpacker
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.parseAs
+import okhttp3.FormBody
+import okhttp3.Request
+import okhttp3.Response
+
+// Rumanhua legacy preference:
+// const val APP_CUSTOMIZATION_URL = "APP_CUSTOMIZATION_URL"
+
+/** 漫漫看 */
+open class MMLook(
+    override val name: String,
+    override val baseUrl: String,
+    private val desktopUrl: String,
+    private val useLegacyMangaUrl: Boolean,
+) : HttpSource() {
+    override val lang: String get() = "zh"
+    override val supportsLatest: Boolean get() = true
+
+    override val client = super.client.newBuilder()
```

Comment:
```suggestion
    override val client = network.cloudflareClient.newBuilder()
```
forks like j2k still don't default to cloudflare client

Reply:
The sites are not using Cloudflare services.

Reply:
since other apps default to cloduflare client, this is for behavioral consistency